bakerydemo/static/css/main.css

```diff
@@ -868,3 +1092,28 @@ span.outline {
     display: none;
   }
 }
+
+/* From Wagtail core */
+/* Responsive image/video classes */
+.rich-text img {
+    max-width: 100%;
+    height: auto;
+}
+.richtext-image.left{
+    float:left;
+}
+.richtext-image.right{
+    float:right;
+}
+.responsive-object {
+    position: relative;
+}
+.responsive-object iframe,
+.responsive-object object,
+.responsive-object embed {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+}
```

Comment:
There might be a mix since I've been a little hurried. I feel like 2-space is the more common convention for CSS/ SASS, but fairly agnostic. I think it might make sense to amend within the CSS refactoring ticket in any case. There's quite a lot of repetition in that file.